projects/addon-doc/src/components/tab-content/tab-content.component.ts

```diff
@@ -0,0 +1,19 @@
+import {ChangeDetectionStrategy, Component, Inject, Input} from '@angular/core';
+import {TUI_IS_CYPRESS} from '@taiga-ui/cdk';
+
+@Component({
+    selector: 'tui-tab-content',
+    templateUrl: './tab-content.template.html',
+    styleUrls: ['./tab-content.template.less'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class TuiTabContentComponent {
+    @Input() isActiveTab: boolean = false;
```

Comment:
Unnecessary trivial type. I thought it would even highlight it as lint error, but maybe that was just TSLint :)
And next line can be `code = ''`, no need to optional complication.